sdk/keyvault/keyvault-certificates/src/identifier.ts

```diff
@@ -4,26 +4,19 @@
 import { parseKeyvaultIdentifier } from "./generated/utils";
 
 /**
- * Valid collection names for Key Vault Certificate identifiers.
+ * Represents different components of a Key Vault Certificate Id.
  */
-export type KeyVaultCertificatesIdentifierCollectionName = "certificates" | "deletedcertificates";
-
-/**
- * Represents a Key Vault identifier and its parsed contents.
- */
-export interface ParsedKeyVaultCertificatesIdentifier {
-  /**
-   * The type of resource under Key Vault that this identifier is referring to.
-   */
-  collection: KeyVaultCertificatesIdentifierCollectionName;
-
+export interface KeyVaultCertificateId {
   /**
-   * The originally received identifier.
+   * The complete representation of the Key Vault Certificate Id. An example could be:
+   *
+   *   https://<keyvault-name>.vault.azure.net/certificates/<certificate-name>/<unique-version-id>
+   *
    */
-  id: string;
+  sourceId: string;
 
   /**
-   * The Key Vault Certificate unique identifier (a URL).
+   * The URL of the Azure Key Vault. Part of the Certificate Id.
```

Comment:
Everything in here is part of the certificate id, so no need to mention that

```suggestion
   * The URL of the Azure Key Vault to which the certificate belongs.
```